Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/Network/NativeWebSocketTransport.cs

```diff
@@ -277,7 +278,7 @@ private async void OnMessageReceived(byte[] bytes)
 			}
 		}
 
-		private async UniTask ProcessApplicationPause(bool pauseStatus)
+		private async Task ProcessApplicationPause(bool pauseStatus)
```

Comment:
We can use unitask for these methods since we are already using it as a dependency

Reply:
I agree, but this will require changing it in many places at once as a chain reaction, shall we do it?